C-Sharp/VectorDistanceCalculation/PE11VectorDistanceCalculation/App.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
     <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.2" />
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.2" /> <!--Why are you guys still using .net framework, .net core and .net 5 is the way to go. Always use the latest and greatest until you cannont.-->
```

Comment:
Good copy! I think .NET Framework has more libraries than .NET Core, so I use the former when I want to get a little fancy (such as adding sounds)...

Reply:
It does not have more libraries. MSFT is not continuing development on .NET Framework. 100% of future projects should be in .net core 3.1 at minimum but should be .net 5. There are some constraints with .net 5.

Reply:
WILCO!